origin-growth/src/scripts/verifyEvents.js

```diff
@@ -0,0 +1,83 @@
+// Scans growth events and runs heuristics to detect fraudulent activity.
+// Mark growth_event rows as either 'Verified' or 'Fraud'.
+
+'use strict'
+
+const Logger = require('logplease')
+const Sequelize = require('sequelize')
+
+const enums = require('../enums')
+const db = require('../models')
+const parseArgv = require('../util/args')
+
+Logger.setLogLevel(process.env.LOG_LEVEL || 'INFO')
+const logger = Logger.create('verifyEvents', { showTimestamp: false })
+
+class VerifyEvents {
+  constructor(config) {
+    this.config = config
+    this.stats = {
+      numProcessed: 0,
+      numVerified: 0,
+      numFraud: 0
+    }
+  }
+
+  // TODO(franck): IMPLEMENT ME
+  _isFraud(event) {
+    return false
+  }
+
+  async process() {
+    const now = new Date()
+
+    // Look for events with status 'Logged'
+    const events = await db.GrowthEvent.findAll({
+      where: {
+        status: enums.GrowthEventStatuses.Logged,
+        createdAt: { [Sequelize.Op.lt]: now }
```

Comment:
Not sure I understand why events with `createdAt` equaling to now are queried?

Reply:
It is useless for now. I was thinking in future I may want to have some delay (to make fraud detection more effective ?) but it is not needed right now. Removed for now and will reconsider when I implement the PR for fraud detection.

Reply:
Ah I missed the `Sequelize.Op.lt` part. Yeah it totally makes sense.